cns/nmagent/client.go

```diff
@@ -84,11 +93,18 @@ func JoinNetwork(networkID, joinNetworkURL string) (*http.Response, error) {
 }
 
 // PublishNetworkContainer publishes given network container
-func PublishNetworkContainer(networkContainerID, createNetworkContainerURL string, requestBodyData []byte) (*http.Response, error) {
+func PublishNetworkContainer(networkContainerID, associatedInterfaceID, accessToken string, requestBodyData []byte) (*http.Response, error) {
 	logger.Printf("[NMAgentClient] PublishNetworkContainer NC: %s", networkContainerID)
 
+	createNCURL := fmt.Sprintf(
+		PutNetworkContainerUrlFmt,
+		WireserverIP,
```

Comment:
With this we are going with the wireserver IP that is in CNS, let me know if you think that isn't best.